draft-ietf-quic-recovery.md

```diff
@@ -908,12 +908,10 @@ Or, expressed as an inter-packet interval:
 interval = smoothed_rtt * packet_size / congestion_window / N
 ~~~
 
-Using a value for `N` that is small, but at least 1 (for example, 1.25) ensures
-that variations in round-trip time don't result in under-utilization of the
-congestion window.  Values of 'N' larger than 1 ultimately result in sending
-packets as acknowledgments are received rather than when timers fire, provided
-the congestion window is fully utilized and acknowledgments arrive at regular
-intervals.
+Using a value for N that is low, but at least 1 (for example, 1.25) ensures that
+variations in round-trip time don't result in under-utilization of the
+congestion window. Note that a higher N results in the inter-packet interval
+becoming smaller, and a large enough N defeats the purpose of pacing.
```

Comment:
This last point is true only in the extreme.

I'm not a fan of this PR, but you probably already know that.